_translator-files/po/sv-SE/Getting-Started.po

```diff
@@ -160,36 +171,40 @@ msgid ""
 "  <figcaption>Let people know who you are</figcaption>\n"
 "</figure>\n"
 msgstr ""
+"<figure>\n"
+"  <img src=\"{% include img/sv-SE-screenshots/settings-profile.inc %}\" loading=\"lazy\" alt=\"Screenshot\" style=\"border: 5px solid grey;\">\n"
+"  <figcaption>Låt folk veta vem du är</figcaption>\n"
+"</figure>\n"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:56
 #, markdown-text
 msgid "Fill in at least “Alias/Name” and close the window."
-msgstr ""
+msgstr "Fyll i minst \"Alias/Namn\" och stäng fönstret."
 
 #. type: Title ##
 #: ../wiki/en/Getting-Started.md:57
 #, markdown-text, no-wrap
 msgid "Connecting to a Server and testing your sound"
-msgstr ""
+msgstr "Ansluta till en server och testa ditt ljud"
 
 #. type: Plain text
 #: ../wiki/en/Getting-Started.md:60
 #, markdown-text
 msgid "Before you connect to a Server, you should **not** be able to hear yourself.  Listening to your sound from Jamulus (and not from yourself) is called “The Golden Rule” and enables you to play in time with others ([see the FAQ](/wiki/FAQ#why-should-i-not-listen-to-my-own-signal))."
-msgstr ""
+msgstr "Innan du ansluter till en server bör du **inte** kunna höra dig själv. Att lyssna på ditt ljud från Jamulus (och inte direkt från dig själv) kallas för \"Den gyllene regeln\" och gör det möjligt för dig att spela i takt med andra ([se FAQ](/wiki/FAQ#why-should-i-not-listen-to-my-own-signal))."
```

Comment:
Translate text after # in link.

Reply:
`varför-bör-jag-inte-lyssna-på-mitt-eget-ljud`